src/org/skypro/skyshop/basket/ProductBasket.java

```diff
@@ -74,9 +88,8 @@ public boolean checkInBasket(String productName) {
     }
 
     public void emptyBasket() {
-        Arrays.fill(products, null);
+        Collections.fill(products, null);
```

Comment:
C коллекциями так нельзя. Коллекции нужно чистить, а не заполнять нулами

```suggestion
        products.clear();
```